assets/css/blueprint.css

```diff
@@ -1,12 +1,5 @@
 @charset "UTF-8";
 
-@import "components/hero.css";
-@import "theme/padding.css";
-@import "theme/text-styles.css";
-@import "theme/margin.css";
-@import "theme/layout.css";
-@import "theme/background.css";
```

Comment:
are there lines in blueprint which depend on these imports?

Reply:
no - these were introduced by me for theming but zj found a way to use scss so i'm swapping over (in tHis PR)